awesomeConfig.cmake

```diff
@@ -329,19 +328,25 @@ if(GENERATE_DOC)
 endif()
 
 # {{{ Configure files
```

Comment:
Changes here are hard to understand.
Should have some comment(s), and I dislike using "DO_COVERAGE" here, but it cannot be avoided probably.